README.md

```diff
@@ -24,6 +24,24 @@ If you have ideas, requests, or proposals for future features, please
 don’t hesitate to open Github issues.
 
 
+## Minimal Supported Rust Versions
+
+We are trying to maintain a decent balance between being able to use older
+compiler versions and using new features that improve safety and usability
+of the crate.
+
+### Minimum version: 1.68.2
+
+The `rust-version` given in `Cargo.toml` is the oldest version that can
+be used to compile the crate with the minimum versions of all dependencies
+selected.
+
```

Comment:
One thing I would add to the documentation is that the command 'cargo +nightly update -Z minimal-versions' gives the minimum version of the dependencies. That might be helpful for people trying to compile using an older version.